provisioning/dashboards/postgres.json

```diff
@@ -204,11 +204,11 @@
         "mediaSources": [
           {
             "field": "concat",
-            "id": "9b9286f8-4d82-4131-a8dd-00f5c0a660d6",
+            "id": "image-2-5-2-0-ver",
```

Comment:
Why there are hardcoded id in multiple datasources?

Reply:
provisioning updated